docs/how-to-guides/themes/theme-json.md

```diff
@@ -1,8 +1,10 @@
-# Themes & Block Editor: experimental theme.json
+# Global Settings (theme.json)
```

Comment:
I still struggle to communicate `theme.json` is both for settings & styles. Perhaps naming this "Global Settings & Styles" would help in communicating that?

To clarify: given this doesn't have any effects in URLs, etc. and can be reverted/updated/etc, I'm fine experimenting with anything (this is not a blocker for me).

Reply:
I read it as styles is one of the types of Global settings you can set, but not sure if that is too nuanced.

I'm open to suggestions, but do think we should name the file a single thing: "Global Properties" ? 🤷‍♂️